src/ui/server/content-strings/fields/insurance/policy-and-exports/index.ts

```diff
@@ -110,7 +110,7 @@ export const POLICY_AND_EXPORTS_FIELDS = {
           NEED_MORE_COVER: 'If you need cover for more than £499,999, ',
           FILL_IN_FORM: {
             TEXT: 'fill in this form instead.',
-            HREF: LINKS.EXTERNAL.FULL_APPLICATION,
+            HREF: LINKS.EXTERNAL.PROPOSAL_FORM,
```

Comment:
Is there a test which would could test the href links to the right form?

Reply:
There are E2E tests for this, but they test the value in field content strings (what's been changed here), which makes sense IMO.

We could update the E2E test to check the value from `LINKS` instead of field content strings. What do you think?

Reply:
yes, maybe we could check the attr.href links tot he right place.  I think I did so in one of my e2e tests